Assignment_3/Assignment_3.1/main.cpp

```diff
@@ -25,7 +25,7 @@ int main(){
     return 0;
 }
 
-string remakeString(string& str, const string& weirdWord){
+string remakeString(const string& str, const string& weirdWord){
```

Comment:
You are not change str, so might as well make it const. Also, if you did then, per convention, const arguments should be first, followed by those that are expected to be modified.